lib/models/application.coffee

```diff
@@ -274,10 +248,11 @@ exports.remove = (name, callback) ->
 ###
 exports.restart = (name, callback) ->
 	exports.get(name).then (application) ->
-		return request.requestAsync
+		return request.send
 			method: 'POST'
 			url: "/application/#{application.id}/restart"
-	.nodeify(_.unary(callback))
+	.return(undefined)
```

Comment:
To prevent resolving the promise with the request response, just as a way to avoid polluting the API with unnecessary information.